ProjectWithUMLDiagram/src/com/beyond/person/entrypoint/Main.java

```diff
@@ -0,0 +1,73 @@
+package com.beyond.person.entrypoint;
+
+import com.beyond.person.core.exceptions.*;
+import com.beyond.person.core.helper.UserInputArguments;
+import com.beyond.person.core.model.BasePerson;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        int count = 0;
```

Comment:
@AraChilingaryan what is this "LAVASH" , we talked about small methods and now this ? change

Reply:
done